app/services/events/stores/aggregated_clickhouse_store.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+module Events
+  module Stores
+    class AggregatedClickhouseStore < ClickhouseStore
+      NIL_GROUP_VALUE = "<nil>"
+
+      def events(force_from: false, ordered: false)
+        # TODO(pre-aggregation): Implement
+      end
+
+      def aggregated_events_sql(force_from: false, ordered: false, select: aggregated_arel_table[Arel.star])
+        query = aggregated_arel_table.where(
+          aggregated_arel_table[:subscription_id].eq(subscription.id)
+            .and(aggregated_arel_table[:organization_id].eq(subscription.organization_id)
+            .and(aggregated_arel_table[:charge_id].eq(charge_id)
+            .and(aggregated_arel_table[:charge_filter_id].eq(charge_filter_id || ""))))
```

Comment:
I'm trying to understand this. Does `charge_filter_id` defaults to `""` in `events_enriched_expanded` ? 

Also from where is `events_enriched_expanded` populated ?